wp-github-sync.php

```diff
@@ -253,3 +253,9 @@ public static function write_log($msg, $write = 'line') {
 }
 
 $wpghs = new WordPress_GitHub_Sync;
+
+function get_the_github_url() {
```

Comment:
What do you think about `get_the_github_url` and `get_the_github_edit_url`? Do you think people would link to "view on GitHub" as opposed to "Edit on GitHub"?